nav2_sms_behavior/package.xml

```diff
@@ -25,15 +25,17 @@
   <depend>builtin_interfaces</depend>
   <depend>tf2_ros</depend>
   <depend>nav2_costmap_2d</depend>
-  <depend>nav2_recoveries</depend>
+  <depend>nav2_behaviors</depend>
   <depend>nav2_core</depend>
   <depend>pluginlib</depend>
+  <depend>action_msgs</depend>
+  <depend>libcurl</depend>
```

Comment:
This doesn't seem like the right rosdep key https://github.com/ros/rosdistro/blob/4f287cd19d8c3127b444ecfca0a0264569aa71de/rosdep/base.yaml#L691

Reply:
Can you give me some advice on how to change it?

Reply:
Correct